src/uikit.cs

```diff
@@ -6106,12 +6106,18 @@ interface UIGestureRecognizerDelegate {
 	[BaseType (typeof(NSObject))]
 	interface UIGraphicsRendererFormat : NSCopying
 	{
+		[Deprecated (PlatformName.TvOS, 11, 0, message: "Use 'PreferredFormat' instead.")]
 		[Static]
 		[Export ("defaultFormat")]
 		UIGraphicsRendererFormat DefaultFormat { get; }
 	
 		[Export ("bounds")]
 		CGRect Bounds { get; }
+
+		[Watch (4,0), TV (11,0), iOS (11,0)]
```

Comment:
is that type included in watchOS ?

Reply:
not it's not in watchOS headers, let's remove it to avoid confusion

Reply:
Yea and actually all that code is under a `#if !WATCH` I didn't see...